lit-config.js

```diff
@@ -46,23 +46,30 @@ module.exports = {
 	"overrides": [{
 		"files": "./**/lang/*.js",	
 		"rules": {
-			"quotes": 0
+			"quotes": "off",
+			"key-spacing": "error",
+			"quote-props": ["error", "consistent"],
```

Comment:
Seems like this rule is deprecated (although so is `quotes`). Are we worried about that?

Reply:
Looks like we're supposed to migrate to [this plugin](https://eslint.style/packages/js) for some of our rules, which we'd probably want to do separate from this

Reply:
Good catch. I'm just going to pull all these out and we can re-assess the language-specific stuff later.